cs247_test.md

```diff
@@ -26,4 +26,4 @@ It outputs the diff (if any) or memory errors (if any) as well as indicator of t
 
 **Note**: You can optionally pass in the specific question `./test.sh 1` to only run tests for that question. You can also specify the test to run for a specific question `./test.sh 1 test.in`, which will only run test.in against a1q1.
 
-**Additional Note:** You can make the script test a2/, a3/, etc. by modifying the variable `i` in `test.sh`. 
+**Additional Note:** You can make the script test a2/, a3/ by modifying the variable `i` in `test.sh`. 
```

Comment:
good catch @tpinto7

Reply:
I agree excellent eye, lgtm2 @ShehryarX

Reply:
Just wondering if any tests have been written for this change? Would give me some confidence to see this feature rigorously tested.

Reply:
@rafitj Good idea, what are your thoughts @tpinto7?

Reply:
@ShehryarX it might be better to wait on the sole contributor of the script, Mufeez, to have final say on this decisive decision.